src/flaskr/auth.py

```diff
@@ -10,5 +10,28 @@
 
 @bp.route('/login', methods=('GET', 'POST'))
 def login():
-    #Functionality goes here
```

Comment:
Please add a unit test case for this functionality. Create a new branch under "tests" for "unit tests", and under that, name your unit test based on the flask functionality. This way I can review your code function, based on the output of the unit test case code. Make sure it is calling on the appropriate file path for this functionality. Thanks! :)